packages/user-operation-controller/src/utils/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { toHex } from '@metamask/controller-utils';
+
+/**
+ * Normalize the given gas value to a string.
+ *
+ * @param gasValue - The gas value to normalize.
+ * @returns The normalized gas value.
+ */
+export function normalizeGasValue(gasValue: string | number): string {
```

Comment:
In the past we've tried to avoid a generic `utils` as it's easier for it to become a catch all.

Would this be better placed within the `PendingUserOperationTracker` as a standard private since it has no other uses yet and is specific to how the tracker to wants to normalize the values?

Reply:
Done in https://github.com/MetaMask/core/pull/4161/commits/1e28a31da243f52098c5ce4536e905848a3f04b1